eng/pipelines/templates/jobs/workload-build.yml

```diff
@@ -0,0 +1,122 @@
+  # If this list needs to be updated, the RegEx in the script also needs to be updated.
+  # The hardcoded list here is necessary as these types need to be available during compilation of the pipeline.
+  # The workload types become separate variables in the create-workload-drops.ps1 and this list is looped over to create multiple drop tasks.
+  workloadDropTypes:
+  - components
+  - packs
+  - precomponents
+
+jobs:
+# Note: jobs.yml automatically includes the Publish Assets job for pushing the artifacts to DARC.
+- template: /eng/common/templates-official/jobs/jobs.yml
+  parameters:
+    enableMicrobuild: true
+    enablePublishBuildAssets: true
+    enableTelemetry: true
+    enablePublishUsingPipelines: true
+    publishAssetsImmediately: true
+    enableSbom: true
+    artifacts:
+      publish:
+        artifacts: true
+        logs: true
+        manifests: true
+    jobs:
+    - job: buildRepo
+      displayName: Build Repo
+      timeoutInMinutes: 120
+      pool:
+        name: $(DncEngInternalBuildPool)
+        image: 1es-windows-2022
+        os: windows
+      steps:
+      - ${{ if eq(parameters.createVSInsertion, true) }}:
+        - task: AzureCLI@2
+          displayName: 🟣 Download workloads for VS insertion
+          inputs:
+            # TODO: Need our own subscription
+            azureSubscription: DotNetStaging
+            scriptType: pscore
+            scriptPath: $(Build.SourcesDirectory)/eng/download-workloads.ps1
+            # Note: The second $ for usePreComponents allows the value to resolve as `$true` or `$false`.
+            arguments: >-
+              -workloadPath '$(Build.SourcesDirectory)/artifacts/workloads'
+              -gitHubPat (ConvertTo-SecureString -String '$(BotAccount-dotnet-bot-repo-PAT)' -AsPlainText -Force)
+              -azDOPat (ConvertTo-SecureString -String '$(dn-bot-all-drop-rw-code-rw-release-all)' -AsPlainText -Force)
+              -workloadListJson '${{ convertToJson(parameters.workloadDropNames) }}'
+              -usePreComponents:$${{ parameters.usePreComponentsForVSInsertion }}
+
+      # https://github.com/dotnet/arcade/blob/ccae251ef033746eb0213329953f5e3c1687693b/Documentation/CorePackages/Publishing.md#basic-onboarding-scenario-for-new-repositories-to-the-current-publishing-version-v3
+      - powershell: >-
+          eng/common/build.ps1
+          -restore -build -sign -pack -publish -ci -msbuildEngine vs
+          -configuration $(_BuildConfig)
+          /p:CreateVSInsertion=${{ parameters.createVSInsertion }}
+          /p:WorkloadDirectory=$(Build.SourcesDirectory)/artifacts/workloads
+          /p:DotNetSignType=$(_SignType)
+          /p:TeamName=$(_TeamName)
+          /p:DotNetPublishUsingPipelines=true
+          /p:OfficialBuildId=$(Build.BuildNumber)
+          /p:StabilizePackageVersion=${{ parameters.stabilizePackageVersion }}
+        displayName: 🟣 Build solution
```

Comment:
this is the spot we'd add an additional build if we wanted to do the other bands from the same branch. We'd set CreateVSInsertion to false in the later builds and it might just work even though it's in the same agent. But per offline conversation, we need to determine if we're inserting workload sets into VS as that complicates things.

Reply:
I'm still very hesitant on running the entire Arcade build process more than once, on the same solution, without clearing the entire output folder structure and other build files. But obviously, we can't do that or then we can't do insertions. Let's keep this on the backburner until we figure out our VS workload set (potential) insertion.